### ISSUE TITLE
(eval):1: bad assignment

### ISSUE THREAD

**Issue by pszypowicz** [eendroroy/alien-minimal#31]
**Describe the bug**
When I enable the theme, zsh starts to throw: 

```
(eval):1: bad assignment
```

**To Reproduce**
1. Starts new shell, error shows, before prompt. Will happen also on empty line submit. 

**Expected behavior**
No errors thrown

**Environment:**
- Plugin Manager: oh my zsh
- ZSH Version: zsh 5.8 (x86_64-ubuntu-linux-gnu)
- Operating System: Windows 10, build 19041.388
- Terminal Emulator: Windows Terminal, CMD, 

**Additional context**
Not sure what causing it, and no more ideas how to debug. When I used other themes, everything worked. When I switch back to old one, everything works. 

I installed the theme with: 
```
git clone --recurse-submodules https://github.com/eendroroy/alien-minimal.git ${ZSH_CUSTOM}/themes/alien-minimal
```

I cleaned the .zshrc from everything, except:

```
export ZSH="/home/user/.oh-my-zsh"
ZSH_THEME="alien-minimal/alien-minimal"

source $ZSH/oh-my-zsh.sh
```

and it still unfortunately happen.

I expect it to be layer 8 problem, i'm just out of ideas how to debug it.

**Comment by eendroroy** [comment]
This might happen from this line: https://github.com/eendroroy/alien-minimal/blob/master/modules/prompt.zsh#L9

Probably one of ENV variables causing the issue while copying it to async process.

You can try adding a `echo` like below:
```bash
__import_env() {
  echo "${1}" | while read -r line; do

    echo "DEBUGGING => ${line} <=" # DEBUG LINE

    eval "export $(echo "${line}" | awk -F'=' '{print $1"=\""$2"\""}')" > /dev/null
  done
}
```

**Comment by pszypowicz** [comment]
So the output is:

```bash
(eval):1: bad assignment                                                                                                                                 
DEBUGGING => <=
```

**Comment by eendroroy** [comment]
Could you please share the output of:
```bash
env | grep --color=never "ENV_VERSION=\|^VIRTUAL_ENV=\|^AM_"
```

BTW, what procedures did you follow to install zsh in windows? If you could share I can try it out in a windows vm.

**Comment by pszypowicz** [comment]
```bash
$ env | grep --color=never "ENV_VERSION=\|^VIRTUAL_ENV=\|^AM_"
(eval):1: bad assignment
DEBUGGING => <=
```

So actually nothing :)

also my .amrc does not exist yet. 

I'm using `WSL 2`, which is implemented in Windows as hyperv vm.

**Comment by pszypowicz** [comment]
I just checked the proposed code and it worked. Thanks!

**Comment by j0ng4b** [comment]
You're welcome ☺️

**Comment by eendroroy** [comment]
Merged #32
